lib/mongo/collection/view/writable.rb

```diff
@@ -55,6 +56,10 @@ def find_one_and_delete
         # @option opts [ true, false ] :upsert Whether to upsert if the document doesn't exist.
         # @option opts [ true, false ] :bypass_document_validation Whether or
         #   not to skip document level validation.
+        # @option opts [ true, false ] :upsert Whether to upsert if the
+        #   document doesn't exist.
```

Comment:
Yeah, I can't figure out why it's in there. Your branch looks up-to-date with master.